R/utils.R

```diff
@@ -37,11 +39,11 @@ csv_reader <- function(file, verbose = FALSE, ...) {
   }
   if (verbose) {
     message("Downloading data from ", file)
-    data <- read_csv_fun(file, ..., guess_max = 500)
+    data <- read_csv_fun(file, ..., guess_max = guess_max)
```

Comment:
Something odd happening with history here...

Reply:
Yeah, I'mnot sure how to fix it as I don't fully know what Iv'e done to break it